### DISCUSSION TITLE
is there an sample app from the avalonia official site?

### DISCUSSION THREAD

**Discussion by charleypeng** [AvaloniaUI/Avalonia#16160] [General]
i saw that avalonia has a perfectly designed app(which seems to be ios), is it a sample app? or what it is called? thanks
<img width="1095" alt="2024-06-28 20 43 23" src="https://github.com/AvaloniaUI/Avalonia/assets/2144710/10c06d80-f726-4636-ae51-f2613a209efc">

**Comment by maxkatz6** [comment]
It's CherylUI theme demo https://github.com/kikipoulet/CherylUI

**Reply by MikeCodesDotNET** [reply]
And the second app is one that @jmacato made for us and we've not released (yet).

**Comment by charleypeng** [comment]
> It's CherylUI theme demo https://github.com/kikipoulet/CherylUI

thanks
